actions/code-release/action.yml

```diff
@@ -124,3 +124,11 @@ runs:
           AWS_SECRET_ACCESS_KEY: ${{inputs.secret-key-production}}
           SOURCE_DIR: './${{inputs.source-directory}}'
           DEST_DIR: '${{inputs.s3-layer-folder}}'
+    - name: Lambda code deploy to staging
+      uses: appleboy/lambda-action@master
+      with:
+        aws_access_key_id: ${{inputs.access-key-staging}}
+        aws_secret_access_key: ${{inputs.secret-key-staging}}
+        aws_region: eu-west-2
+        function_name: ${{inputs.repository-name}}
```

Comment:
I think this should be a new variable `function-name` as repository name is slightly different.

Reply:
fixed it with one new step